### PR TITLE
ENH uncache errors run

### PR DESCRIPTION
**Pull request by juAlberge**

### CODE REVIEW

File: benchopt/benchmark.py [hunk 1]
```diff
@@ -357,6 +357,16 @@ def _func_cached(**kwargs):
 
         return _func_cached
 
+    def clear_cache(self, func, ignore=None, **kwargs):
+        """Clear the cache for the given function."""
+
+        # Create a cached function the computations in the benchmark folder
```
Comment (mathurinm) [inline benchopt/benchmark.py:363]: @juAlberge could you clarify this comment?

**Review by mathurinm — APPROVED** [review]

File: doc/names.inc [hunk 2]
```diff
@@ -27,3 +27,7 @@
 .. _Christopher Marouani: https://github.com/chris-mrn
 
 .. _Pierre-Antoine Comby: https://github.com/paquiteau
+
+.. _Virginie Loison: https://www.linkedin.com/in/virginie-loison/
+
+.. _Julie Alberge: https://www.linkedin.com/in/julie-alberge
```
Comment (tomMoral) [inline doc/names.inc:33]:
```suggestion
.. _Julie Alberge: https://www.linkedin.com/in/julie-alberge

```

File: README.rst [hunk 3]
```diff
@@ -44,7 +44,8 @@ It is also possible to use the **latest development version**. To do so, run ins
 
 .. code-block:: bash
 
-    pip install --pre benchopt -U -i https://test.pypi.org/simple
+    git clone https://github.com/benchopt/benchopt.git 
+    pip install -e .
```
Comment (tomMoral) [inline README.rst:48]:
```suggestion
    pip install git+https://github.com/benchopt/benchopt.git
```

**Review by tomMoral — APPROVED** [review]

**Comment by mathurinm** [comment]
Thanks @vloison and @juAlberge !

**Pull request merged**